roxctl/scanner/uploaddb/uploaddb_test.go

```diff
@@ -0,0 +1,20 @@
+package uploaddb
+
+import (
+	"io/fs"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestScannerUploadDbCommandFail(t *testing.T) {
+	cmdNoFile := scannerUploadDbCommand{filename: "non-existing-filename"}
+
+	actualErr := cmdNoFile.uploadDd()
+
+	require.Error(t, actualErr)
+	assert.ErrorIs(t, errors.Cause(actualErr), fs.ErrNotExist)
```

Comment:
```suggestion
	assert.ErrorIs(t, actualErr, fs.ErrNotExist)
```
This shall work without `.Cause()`. `ErrorIs()` relies to `Unwrap()` inside which is equivalent to `Cause()`. Moreover, we shall not depend on any knowledge on whether a specific interface is being supported: `uploadDd()` returns just `error` which IIRC is not guaranteed to implement `Cause()` or be a type from `pkg/errors`.

Reply:
✅